fix(platform-server): remove peer dependency on animations

crisbeto opened the pull request:
The `@angular/platform-server` package had a peer dependency on `@angular/animations` which wasn't being used anywhere.

alan-agius4 reviewed (approved):
LGTM

alan-agius4 commented:
Nit: might also be worth removing https://github.com/angular/angular/blob/37b6bd5addfbdd9949dfbec825c2c8203d2a076d/packages/platform-server/BUILD.bazel#L18

crisbeto committed:
fix(platform-server): remove peer dependency on animations

The `@angular/platform-server` package had a peer dependency on `@angular/animations` which wasn't being used anywhere.

pkozlowski-opensource commented:
This PR was merged into the repository by commit 1cfbfc66d3a24b6c41abf13550e7c2911e20b550.

The changes were merged into the following branches: main, 19.0.x

davidbusuttil commented:
Now, we are getting:

Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@angular/animations' imported from C:\Users\...\node_modules\@angular\platform-browser\fesm2022\animations.mjs
